app/presenters/contact_presenter.rb

```diff
@@ -38,10 +38,17 @@ def links
     # This means that if the `organisations` in the links is modified, contacts-admin will not
     # accidentally overwrite it - and we can start tagging contact pages with content-tagger.
     # See: https://github.com/alphagov/contacts-admin/pull/231.
+    
+    parent_content_id = contact.organisation.content_id
+
+    if parent_content_id == PublishFinders::HMRC_ORGANISATION_CONTENT_ID
+      parent_content_id = PublishFinders::HMRC_CONTACTS_CONTENT_ID
+    end
```

Comment:
This was a little confusing trying to see the difference between `HMRC_ORGANISATION` and `HMRC_CONTACTS`, and re-defining the same variable a few times. But it depends how you (and the linter) feel about hanging indents. How about:

```
parent_content_id = if contact.organisation.content_id == PublishFinders::HMRC_ORGANISATION_CONTENT_ID
                      PublishFinders::HMRC_CONTACTS_CONTENT_ID
                    else
                      contact.organisation.content_id
                    end
```

Reply:
Yup - I've moved it into a private method to further help with readability.